seqr/views/utils/airtable_utils.py

```diff
@@ -8,7 +8,7 @@
 
 logger = SeqrLogger(__name__)
 
-MAX_OR_FILTERS = 200
+MAX_OR_FILTERS = 270
```

Comment:
I tested with real-world data and this higher cutoff still works for every single seqr project